docs/generated/reference/BUILD.bazel

```diff
@@ -7,7 +7,10 @@ filegroup(
 
 filegroup(
     name = "all-srcs",
-    srcs = [":package-srcs"],
+    srcs = [
+        ":package-srcs",
+        "//docs/generated/reference/openapi:all-srcs",
+    ],
```

Comment:
This change should be reverted automatically once you remove the openapi_generated.go file and re-run update-bazel 😄